packages/protocol/lib/registry-utils.ts

```diff
@@ -1,42 +1,32 @@
-export const attestationsRegistryId: string = 'Attestations'
-export const bondedDepositsRegistryId: string = 'BondedDeposits'
-export const escrowRegistryId: string = 'Escrow'
-export const exchangeRegistryId: string = 'Exchange'
-export const gasCurrencyWhitelistRegistryId: string = 'GasCurrencyWhitelist'
-export const gasPriceMinimumRegistryId: string = 'GasPriceMinimum'
-export const goldTokenRegistryId: string = 'GoldToken'
-export const governanceRegistryId: string = 'Governance'
-export const randomRegistryId = 'Random'
-export const reserveRegistryId: string = 'Reserve'
-export const sortedOraclesRegistryId: string = 'SortedOracles'
-export const stableTokenRegistryId: string = 'StableToken'
-export const validatorsRegistryId: string = 'Validators'
+export enum CeloContract {
```

Comment:
Should this be something like `CeloContractName` to not confuse with functional contract objects?

Reply:
yeah. can be. Since contrackit already uses a `enum CeloContract`. i've used the same name.

I can change it if you'd like

Reply:
changed to `CeloContractName`